pkgs/development/tools/rust/cargo-pgrx/default.nix

```diff
@@ -77,4 +77,11 @@ in
     hash = "sha256-0m9oaqjU42RYyttkTihADDrRMjr2WoK/8sInZALeHws=";
     cargoHash = "sha256-9XTIcpoCnROP63ZTDgMMMmj0kPggiTazKlKQfCgXKzk=";
   };
+
+  cargo-pgrx_0_12_6 = generic {
```

Comment:
Can't we bump the alpha version above?

Reply:
This breaks vector-rs I'll have a look into that

Reply:
Ah, bummer.
Nvm, you don't have to fix it up in this PR.

Reply:
> Should we remove it?

Go for it.
I wouldn't do it on stable, but that's not the case here.

>  I wonder whether we should take responsibility for this with the @NixOS/postgres team as well?

Sounds good. Do you want to file a PR for that?